src/fides/api/task/deprecated_graph_task.py

```diff
@@ -108,7 +110,26 @@ def run_access_request_deprecated(
     session: Session,
 ) -> Dict[str, List[Row]]:
     """Deprecated: Run the access request sequentially in-memory using Dask"""
-    traversal: Traversal = Traversal(graph, identity)
+    try:
+        traversal: Traversal = Traversal(graph, identity)
+    except TraversalError as err:
```

Comment:
Nice tracking this down for DSR 2.0 👍, I have similar concerns with the creation of the execution log